buildspec.yml

```diff
@@ -19,6 +19,33 @@ phases:
         AWS_CONTAINER_CREDENTIALS_RELATIVE_URI= AWS_DEFAULT_REGION=
         tox -e py27,py36 -- tests/unit
 
+      # run notebook test
+      - |
+        if has-matching-changes "tests/" "src/*.py" "docker/*/Dockerfile"; then
```

Comment:
We don't need to run notebooks tests of someone touched 'tests/', do we?

Reply:
good point. i copied the check from the integ test guard. i will narrow it.